sync/landing.py

```diff
@@ -1038,14 +1039,19 @@ def update_landing(git_gecko, git_wpt, prev_wpt_head=None, new_wpt_head=None,
         try_notify_downstream(commits, landing_is_complete=pushed)
 
         if pushed:
-            try:
-                tasks.retrigger.apply_async()
-            except OperationalError:
-                logger.warning("Failed to retrigger blocked syncs")
-
+            retrigger()
     return landing
 
 
+def retrigger():
+    try:
+        # Avoid circular import
+        from . import tasks
+        tasks.retrigger.apply_async()
+    except OperationalError:
+        logger.warning("Failed to retrigger blocked syncs")
```

Comment:
This seems to be completely unrelated to the conversion. Or maybe because of the import?

Reply:
Yeah, we need to avoid the circular import, and it means we need to use a different approach to mocking this in the testing, so this is required to make the tests pass again.